src/components/accordion/Accordion.spec.jsx

```diff
@@ -124,7 +125,7 @@ describe('<Accordion>', () => {
 
     expect(
       accordion
-        .find('Box')
+        .find(Box)
```

Comment:
how about setting Box.displayName = "Box", if I understand correctly it uses displayName to find component without a need of importing real component

Reply:
good point 🤔  Haven't thought about it, but actually this seems like a better solution

Reply:
@Malinicz you can wrap setting display name in env check and only set in on development /tests.

something like: 

```
if(dev) {
...
}

where dev is ofc check for `process.env.NODE_ENV`
```

Reply:
yeah but then I would have the same problem in brainly-frontend, right?

Reply:
and if I just set displayName on component, I also would be able to find by string in our repo, or I'm wrong @coderitual?

Reply:
@Malinicz you should be able to find it in our repo too as long as we don't set production flag for tests in brainly frontend. This code will be present in production build of style guide and only removed on production build of application uses style guide.

Reply:
@Malinicz So setting `export const __DEV__ = process.env.NODE_ENV !== "production"` in some utils file and importing it into component file should work. This also should work with application which uses our component. So in brainly frontend we can test component using strings as well